compiler/rustc_hir/src/hir.rs

```diff
@@ -3479,7 +3491,7 @@ impl<'hir> Node<'hir> {
 #[cfg(all(target_arch = "x86_64", target_pointer_width = "64"))]
 mod size_asserts {
     rustc_data_structures::static_assert_size!(super::Block<'static>, 48);
-    rustc_data_structures::static_assert_size!(super::Expr<'static>, 64);
+    //rustc_data_structures::static_assert_size!(super::Expr<'static>, 64); // FIXME
```

Comment:
Likewise, should we box the `Closure` variant?

Reply:
So like
```rust
    // ExprKind
    Closure(&'hir Closure<'hir>),
```
```rust
struct Closure<'hir> {
    binder: &'hir ClosureBinder,
    capture_clause: CaptureBy,
    bound_generic_params: &'hir [GenericParam<'hir>],
    fn_decl: &'hir FnDecl<'hir>,
    body: BodyId,
    fn_decl_span: Span,
    movability: Option<Movability>,
}
```
?

Maybe, I'm not sure.